dev/tests/static/testsuite/Magento/Test/Integrity/DeclarativeDependencyTest.php

```diff
@@ -131,14 +138,13 @@ private function getErrorMessage(string $id): string
      *
      * @param string $file
      * @return mixed
-     * @throws \Exception
+     * @throws InspectionException
      */
     private function readJsonFile(string $file, bool $asArray = false)
     {
         $decodedJson = json_decode(file_get_contents($file), $asArray);
         if (null == $decodedJson) {
-            //phpcs:ignore Magento2.Exceptions.DirectThrow
-            throw new \Exception("Invalid Json: $file");
+            throw new InspectionException("Invalid Json: $file");
```

Comment:
missed this, we usually do a localized exception that's translatable and any parameters like file go into a "%1" or "%s" if using sprintf. 
This is probably fine because it's a test.  But "Invalid Json: $file" would still be a risky practice and sprintf would be preferred.